__tests__/search.js

```diff
@@ -35,19 +35,77 @@ class MockVFSServiceProvider extends ServiceProvider {
     ];
   }
 }
+class MySearchAdapter {
```

Comment:
The results from this adapter has to be *actually tested*. See https://github.com/os-js/osjs-client/blob/d6781762ce7948a5a9c19e16d1917f715b38de82/__tests__/search.js#L53

Reply:
> 

I tested It was ok

Reply:
> The results from this adapter has to be _actually tested_. See
> 
> https://github.com/os-js/osjs-client/blob/d6781762ce7948a5a9c19e16d1917f715b38de82/__tests__/search.js#L53

I tested It was ok

Reply:
But you're not testing **anything**.